.gitignore

```diff
@@ -0,0 +1,7 @@
+.aider*
+.env
+dist
+.yalc*
+yalc*
+node_modules
+.DS_Store
```

Comment:
`bun init -y` will generate this file a bit better.

Make sure to remove the .DS_Store file that was accidentally committed